R/linear_pool_sample.R

```diff
@@ -0,0 +1,128 @@
+#' Helper function for computing ensemble model outputs as a linear pool
+#' (distributional mixture) of component model outputs for the `sample`
+#' output type.
+#'
+#' @inheritParams linear_pool
+#' @param n_output_samples `numeric` that specifies how many sample forecasts to
+#'   return per unique combination of task IDs. Defaults to NULL, in which case
```

Comment:
```suggestion
#'   return per unique combination of task IDs. Currently the only supported value is NULL, in which case
```

Reply:
But actually, do we need this in the ROxygen docs for this function?  It seems like it would already be covered by the `@inheritParams linear_pool`.